scripts/setup

```diff
@@ -34,9 +34,24 @@ pipenv run invenio index destroy --force --yes-i-know
 pipenv run invenio index init --force
 pipenv run invenio index queue init
 
-# init oaiharvester config from file
-pipenv run invenio oaiharvester initconfig data/oaisources.yml
-# load source NJ
-pipenv run invenio oaiharvester harvest -n NJ -k -q --signals
-# load source VS
-pipenv run invenio oaiharvester harvest -n VS -k -q --signals
+# # init oaiharvester config from file
```

Comment:
do you really want to keep this? Can we remove these comments?

Reply:
removed